clarity/next/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "next-tests",
+  "version": "1.0.0",
+  "description": "Run unit tests on this project.",
+  "private": true,
+  "scripts": {
+    "test": "vitest run",
+    "test:report": "vitest run -- --coverage --costs",
+    "test:watch": "chokidar \"tests/**/*.ts\" \"contracts/**/*.clar\" -c \"npm run test:report\""
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@hirosystems/clarinet-sdk": "^1.1.0",
+    "@stacks/transactions": "^6.9.0",
+    "chokidar-cli": "^3.0.0",
+    "typescript": "^5.2.2",
+    "vite": "^4.4.9",
```

Comment:
fast-check is missing, should be added.

Reply:
What version?

Reply:
In this repo, for romeo, I've been using `"fast-check": "^3.13.1"` with no issues.

Reply:
Included!